### PR TITLE
POM-374 Remove my-account/ad route as it requires authentication and complicates URL sharing

### PR DESCRIPTION
**Pull request by felix1313**

### CODE REVIEW

File: src/app/shared/components/category-navigation/category-navigation.component.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export class CategoryNavigationComponent {
   constructor(private router: Router) {}
 
   activeRoute(): string | undefined {
-    return this.router.url.split('/')[2];
+    return this.router.url.split('?')[0].split('/')[2];
```
Comment (bbartosiak) [inline src/app/shared/components/category-navigation/category-navigation.component.ts:32]: a może brać tylko drugi path ? Jak było wcześniej
this.router.url.split('/')[2],
/znajdz-pomoc/**noclegi**
/dodaj-ogloszenie/**noclegi**

**Review by bbartosiak — APPROVED** [review]

File: src/app/shared/components/category-navigation/category-navigation.component.ts (hunk 1 above)
Comment (bbartosiak) [inline src/app/shared/components/category-navigation/category-navigation.component.ts:32]: zawsze mamy w url '?'

Comment (bbartosiak) [reply src/app/shared/components/category-navigation/category-navigation.component.ts:32]: może to będzie bardziej kompleksowe:
return !!Object.values(CategoryRoutingName).filter(el => this.router.url.includes(el));

Comment (felix1313) [reply src/app/shared/components/category-navigation/category-navigation.component.ts:32]: If we don't have '?', split returns an array with single value - original string. So this part looks safe.

Your idea makes sense as well. I can change it.

**Review by bbartosiak — APPROVED** [review]

**Pull request merged**